recipes/gfal2-util/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "gfal2-util" %}
+{% set version = "1.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/dmc/gfal2-util/-/archive/v{{ version }}/gfal2-util-v{{ version }}.tar.gz
+  sha256: d6bd5aae8117ff7b14a87d76f4531016b4e600170bafaa8e9d8cbe9f759e04f7
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - rm -rf src  # [not py2k]
+    - mv src_py3 src  # [not py2k]
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - python-gfal2
+
+test:
+  # source_files:
```

Comment:
The test suite doesn't work due to bugs (it expects to run in the source directory without having been installed). I'll just leave it commented for now.